docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -248,4 +248,18 @@ If the problem persists, then please contact Couchbase Technical Support.
 
 *Further Reading*: xref:7.0@server:metrics-reference:data-service-metrics.adoc#kv_ep_storage_age_second[Data Service Metrics], xref:7.0@server:install:sizing-general.adoc#sizing-data-service-nodes[Sizing Data Service Nodes]
 
+[#CB90071]
+=== Compaction Failure (CB90071)
+
+*Background*: Auto-Compaction can be set to run either at a specific time, or when bucket storage becomes too low.
+Auto-Compaction requires some free space in order to run, if there is none available then compaction cannot run.
```

Comment:
if -> so if?

Do we have a link to related docs and is there a ballpark calculation of roughly how much (rather than some), e.g., recommended amount to keep free

Reply:
Fixed, and I don't know if we have suggested amounts, I think that's something we'd discuss on a case by case basis.